src/averageAges.js

```diff
@@ -55,7 +76,23 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const parentFilter = people.filter(function(person) {
```

Comment:
Misha recommends to use arrow function instead. You used arrow function everywere, why here not?)

Reply:
last function was the hardest, so i wrote for better understanding for myself)
but i will fix